airbyte-integrations/bases/source-acceptance-test/sample_files/acceptance-test-config.yml

```diff
@@ -13,6 +13,11 @@ tests:
     - config_path: "secrets/config.json"
       configured_catalog_path: "sample_files/configured_catalog.json"
       validate_output_from_all_streams: yes
+      expect_records:
```

Comment:
why not call it `expected_records`? it follows the `expected/actual` naming convention for tests

Reply:
originally it was so, but in the test we have `expected_records` fixture that an actual records we expect, and config class `inputs.expected_records` (`inputs.expected_records.extra_fields`, etc) which might cause confusion.